lib/src/HttpClientImpl.h

```diff
@@ -115,6 +118,14 @@ class HttpClientImpl final : public HttpClient,
         sockOptCallback_ = std::move(cb);
     }
 
+    std::size_t requestsBufferSize() override
+    {
+        std::promise<std::size_t> bufferSize_;
```

Comment:
please use bufferSize as the temporary variable name.

Reply:
done